extensions/micrometer/deployment/src/test/java/io/quarkus/micrometer/deployment/binder/GrpcMetricsDisabledTest.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.micrometer.deployment.binder;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import javax.enterprise.inject.spi.BeanManager;
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class GrpcMetricsDisabledTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .withConfigurationResource("test-logging.properties")
+            .overrideConfigKey("quarkus.micrometer.binder.grpc-client.enabled", "true")
+            .overrideConfigKey("quarkus.micrometer.binder.grpc-server.enabled", "true")
+
+            .overrideConfigKey("quarkus.micrometer.binder-enabled-default", "false")
+            .overrideConfigKey("quarkus.micrometer.registry-enabled-default", "false")
+            .withEmptyApplication();
+
+    @Inject
+    BeanManager beans;
+
+    @Test
+    void testNoInstancePresentIfNoKafkaClientsClass() {
```

Comment:
nit -- but I assume this is a copy-paste method name?

Reply:
Oh no! You are revealing my secret... :-)

Reply:
Ok, we are good, I've hidden my secret...